db/migrate/20230104171944_create_bookings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.datetime :start_date
+      t.datetime :end_date
```

Comment:
Simply rename these 2, no need for another migration just to rename attributes 🙃